### ISSUE TITLE
Спасибо за работу!

### ISSUE THREAD

**Issue by dimabendera** [RegisRivijski/TensorflowJS-RHT#1]
Вы пошли в правильных направлениях,
Один из способов получить более премлевую точность - найти подходящий learning_rate
Но не стоит забывать что уменьшение learning_rate может повлечь за собой более медленную сходимость сети (возможно поэтому вам и пришлось увеличить количество эпох):
<img src="https://www.bigrabbitdata.com/wp-content/uploads/2020/06/learning_rate.png">
Что бы этого избежать есть защитный механизм - ставить большой(в разумных пределах) learning_rate - после каждой эпохи смотреть на loss - и если он не уменшается N эпох то уменшать learning_rate.

Так же вы продублировали слой maxPool2d
https://github.com/RegisRivijski/TensorflowJS-RHT/blob/main/train_model.js#L52
Это не очень хорошая практика. 
После каждого из maxPool2d наша матрица уменшается в размерности - при одном maxPool2d после слоя конволюции это имеет место быть, но два подряд maxPool2d - просто теряют некоторую часть информации.

Спасибо за работу!
